src/website/app/Page.js

```diff
@@ -400,6 +400,10 @@ export default class Page extends Component<Props, State> {
       isNavOpen,
       tabIndex: isNavOpen ? '-1' : undefined
     };
+    const navProps = {
+      currentDemo: pageMeta && pageMeta.title && pageMeta.title.split(' ')[0],
```

Comment:
Instead of this, which is fragile, add a new prop passed to Page in Router, that uses `slug`.